ccmlib/common.py

```diff
@@ -903,17 +906,21 @@ def get_supported_jdk_versions(cassandra_version, install_dir, for_build, env):
         if cassandra_version and cassandra_version >= LooseVersion('5.0'):
             build_versions = [11, 17]
             run_versions = [11, 17]
+            info("Cassandra 5.0+ detected, using Java 11 and 17")
         elif cassandra_version and cassandra_version >= LooseVersion('4.0'):
-            if 'CASSANDRA_USE_JDK11' not in env:
+            if 'CASSANDRA_USE_JDK11' not in env or env['CASSANDRA_USE_JDK11'] != 'true':
```

Comment:
```suggestion
            if 'CASSANDRA_USE_JDK11' not in env or env['CASSANDRA_USE_JDK11'].lower() != 'true':
```

Reply:
build.xml (ant) accepts all of `true`, `yes`, and `on`, case-insensitive…

Reply:
as we talked, instead of doing this, I'm removing the interpretation of this env variable